pkgs/development/python-modules/apprise/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
   nativeBuildInputs = [ Babel ];
 
   propagatedBuildInputs = [
-    requests requests_oauthlib six click markdown pyyaml
+    requests requests_oauthlib six click markdown pyyaml gntp sleekxmpp
   ];
```

Comment:
the two new dependencies are only needed for testing https://github.com/caronc/apprise/blob/f6ce71c65d0c5abfa470978cd719d29e69b497ea/setup.py#L109

please put them in `checkInputs`

Reply:
not so sure that they are only used in tests: https://github.com/caronc/apprise/blob/v0.8.7/apprise/plugins/NotifyXMPP/SleekXmppAdapter.py
https://github.com/caronc/apprise/blob/v0.8.7/apprise/plugins/NotifyGrowl.py#L37

Reply:
That looks optional, and generally they aren't included in the base package in nixpkgs; as there's not a convenient way to replicate the "extras" features of pip.

Reply:
ok no problem, should be fixed